sections/hero.liquid

```diff
@@ -1,7 +1,18 @@
 {% for block in section.blocks %}
-  <h1>{{ block.settings.heading }}</h1>
-  <h2>{{ block.settings.subheading }}</h2>
-  <h3>{{ block.settings.text }}</h3>
+  {% case block.type %}
+    {% when 'heading' %}
+      <div>
+        {{ block.settings.heading }}
+      </div>  
+    {% when 'subheading' %}
+      <div>
+        {{ block.settings.subheading }}
+      </div>
+    {% when 'text' %}
+      <div>
+        {{ block.settings.text }}
+      </div>
+  {% endcase %}
 {% endfor %}
```

Comment:
Cool, but I think we need to have our own wrapper for section content. And use semantic tags for blocks content.